src/platform/imx8/include/platform/lib/memory.h

```diff
@@ -172,7 +172,14 @@
 #define HEAP_BUF_ALIGNMENT		PLATFORM_DCACHE_ALIGN
 
 #if !defined(__ASSEMBLER__) && !defined(LINKER)
+
 void platform_init_memmap(void);
+
+static inline void *platform_shared_get(void *ptr, int bytes)
+{
+	return ptr;
+}
+
```

Comment:
Nice thought put into this commit, indeed it didn't make sense to implement stuff like this on this platform, i.MX doesn't have such an aliased region. I'd already approve but I haven't finished reviewing the series yet.